hack/utils/kops.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -uo pipefail
+
+OS_ARCH=$(go env GOOS)-amd64
+
+kops::install() {
+  INSTALL_PATH=${1}
+  KOPS_VERSION=${2}
+  if [[ ! -e ${INSTALL_PATH}/kops ]]; then
+    KOPS_DOWNLOAD_URL=https://github.com/kubernetes/kops/releases/download/v${KOPS_VERSION}/kops-${OS_ARCH}
+    curl -L -X GET "${KOPS_DOWNLOAD_URL}" -o "${INSTALL_PATH}"/kops
+    chmod +x "${INSTALL_PATH}"/kops
+  fi
```

Comment:
Not crucial, but it would be helpful to log something in the else such as `kops is already installed, not reinstalling.`

Reply:
the curl is not silent, so in a roundabout way it is still possible to tell if the install is being skipped. But yeah, the more logging the better.